### PR TITLE
Include documentation for POOL_SIZE

### PR DESCRIPTION
**Pull request by jeregrine**
https://github.com/phoenixframework/phoenix/issues/1730#event-673287024

cc @josevalim

### CODE REVIEW

File: deployment/E_heroku.md [hunk 1]
```diff
@@ -105,9 +105,23 @@ Then, we'll add the production database configuration to `config/prod.exs`:
 config :hello_phoenix, HelloPhoenix.Repo,
   adapter: Ecto.Adapters.Postgres,
   url: System.get_env("DATABASE_URL"),
-  pool_size: 20,
+  pool_size:String.to_integer(System.get_env("POOL_SIZE") || "10"),
```
Comment (stevedomin) [inline deployment/E_heroku.md]: I think there is a space missing after the colon

Comment (stevedomin) [inline deployment/E_heroku.md]: Sorry my OCD kicked in

Comment (jeregrine) [inline deployment/E_heroku.md]: No need to be sorry, fixed :-)

**Comment by endersstocker** [comment]
Very helpful! 👍

**Pull request merged**